docs/antora.yml

```
@@ -8,3 +8,4 @@ asciidoc:
   attributes:
     product: APPUiO Cloud
     zone: APPUiO Zone
+    idp: APPUiO Keycloak
```

Any preferences here? Other alternative would be `APPUiO IDP` or `APPUiO ID`. Other proposals?

---

```suggestion
    idp: APPUiO IdP
```

Note: The `d` should be lowercase, see https://csrc.nist.gov/glossary/term/Identity_Provider.